packages/corvid-cli/src/apps/open-editor.js

```diff
@@ -20,6 +20,12 @@ const openEditorApp = ({ useSsl = true } = {}) => ({
         console.log(message);
       });
 
+      client.on("parse-error", path => {
+        const message = `Parse Error in file ${path}`;
```

Comment:
we need to rephrase it for users i think

Reply:
@wixysam can you help with rephrasing?

Reply:
How do the files get to this situation? Is there something to do about it?

Reply:
How do the files get to this situation? Is there something to do about it?

Reply:
User actions mostly

Reply:
And how do they fix it? Try to revert to an old version, do a pull, anything else?

Reply:
I want to tag @chaimk for his opinion here, but he is not a member of this organization. Maybe turn around and give him a tap.

Here's what I have so far:

Error parsing ${path}. The file has been corrupted. Try reverting to an older version.

Reply:
ok